### ISSUE TITLE
Mappings from orbit type abbreviations to orbit type complete names (using record in TypeScript)

### ISSUE THREAD

**Issue by jackiefasty** [Bone008/orbiteye#68]

**Comment by jackiefasty** [comment]
Proceed as how was done for owner abbreviation and complete name, also using record.

**Comment by jackiefasty** [comment]
Basing the orbitClass and orbitTypes from https://www.satellitetoday.com/content-collection/ses-hub-geo-meo-and-leo/

**Comment by sahilshahpatel** [comment]
@jakifasty I wouldn't use that source necessarily because that's not where these labels came from. [This](https://s3.amazonaws.com/ucs-documents/nuclear-weapons/sat-database/4-11-17-update/User+Guide+1-1-17+wAppendix.pdf) is the user manual for the UCS database where the labels originate. Use the names here.

The most important difference I noticed is that GEO stands for geosynchronous and NOT geostationary in our data.

EDIT: I looked over the commit and saw that the translation is limited to orbit class and doesn't go into subtypes. In that case, I just had two small suggestions which I left as comments on the commit 9468428839c27efbe586000767f0d8ab81529066
